src/app/item-review/item-review.component.js

```diff
@@ -92,6 +97,20 @@ function ItemReviewController(dimDestinyTrackerService, $scope, $rootScope) {
 
   vm.reviewLabels = [5, 4, 3, 2, 1];
 
+  vm.defs = {};
+
+  getDefinitions().then((defs) => {
```

Comment:
I think this component also gets used for Destiny 1. Can we wrap this in a check that the item is a D2 item to avoid loading the D2 manifest in D1 mode?